database/vmaas_db_postgresql.sql

```diff
@@ -1284,6 +1284,61 @@ CREATE TABLE IF NOT EXISTS csaf_file (
 )TABLESPACE pg_default;
 
 
+-- -----------------------------------------------------
+-- Table vmaas.csaf_product_status
+-- https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#3239-vulnerabilities-property---product-status
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS csaf_product_status (
+    id   INT NOT NULL,
+    name TEXT UNIQUE NOT NULL, CHECK (NOT empty(name)),
+    PRIMARY KEY (id)
+)TABLESPACE pg_default;
+
+INSERT INTO csaf_product_status (id, name)
+VALUES  (1, 'first_affected'),
+        (2, 'first_fixed'),
+        (3, 'fixed'),
+        (4, 'known_affected'),
+        (5, 'known_not_affected'),
+        (6, 'last_affected'),
+        (7, 'recommended'),
+        (8, 'under_investigation')
+ON CONFLICT DO NOTHING;
+
+
+-- -----------------------------------------------------
+-- Table vmaas.csaf_products
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS csaf_products (
+  id             SERIAL,
+  cpe            TEXT NOT NULL, CHECK (NOT empty(cpe)),
+  -- product contains only package_name for known_affected vulnerabilities
+  package_name   TEXT NULL, CHECK (NOT empty(package_name)),
```

Comment:
from `package_name` and `package` columns keep only `package` column for now and store there the package name or the full nevra.